core/modules/system/css/system.theme.css

```diff
@@ -544,3 +544,10 @@ img.align-right {
   display: table;
   clear: both;
 }
+
+/**
+ * Labels & machine names.
+ */
+.machine-name {
+  font-size: smaller;
```

Comment:
Do we use `smaller` elswehere in `system.theme.css`?

Reply:
We use `font-size: smaller;` in 3 files in core:
- views_ui module: https://github.com/backdrop/backdrop/blob/1.x/core/modules/views_ui/css/views_ui.admin.theme.css#L307
- system module: https://github.com/backdrop/backdrop/blob/1.x/core/modules/system/css/system.maintenance.css#L9
- seven theme:
  - https://github.com/backdrop/backdrop/blob/1.x/core/themes/seven/css/seven.base.css#L93
  - https://github.com/backdrop/backdrop/blob/1.x/core/themes/seven/css/seven.base.css#L97
  - https://github.com/backdrop/backdrop/blob/1.x/core/themes/seven/css/seven.base.css#L102

Reply:
@jenlampton I believe that this is the last remaining concern for this PR. Please let me know if you would like that changed to a specific px/em value instead.

Reply:
No, since it's in `/core/modules/system/css/system.maintenance.css` It's a pattern we use in core (and not only in Seven) --- I just hadn't seen it before!